txtorcon/torinfo.py

```diff
@@ -99,6 +109,9 @@ def stripper(key, arg):
 
         d = self.proto.get_info_raw(req)
         d.addCallback(functools.partial(stripper, req))
+        descriptor_keys = ["desc/name", "desc/id"]
+        if self.info_key in descriptor_keys and self.proto.use_stem:
+            d.addCallback(_parse_stem_descriptor)
         return d
```

Comment:
Does this mean that we do not have a _parse_stem_descriptor method and, just directly return the RelayDescriptor object as a callback?